src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+function getEmployees(list) {
+  return Array.from(list).map((item) => {
+    return {
+      name: item.innerText,
+      position: item.getAttribute('data-position'),
+      salary: parseSalary(item.getAttribute('data-salary')),
+      age: parseInt(item.getAttribute('data-age'), 10),
+    };
+  });
+}
+
+function parseSalary(salaryString) {
+  return parseInt(salaryString.replace(/[^0-9.-]+/g, ''), 10);
```

Comment:
The `parseSalary` function uses `parseInt` to convert the salary string to an integer. This might lead to incorrect parsing if the salary includes decimal values. Consider using `parseFloat` instead if the salary can be a floating-point number.